Symfony/CS/Tests/Fixer/Symfony/NewWithBracesFixerTest.php

```diff
@@ -81,6 +81,10 @@ public function provideStandardCases()
                 '<?php $a = new $b[\'class\'] ;',
             ),
             array(
+                "<?php \$a = new \$b['class_x0B']()\r\n\t ; ",
+                "<?php \$a = new \$b['class_x0B']\r\n\t ; " ,
+            ),
```

Comment:
It's already there: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.8/Symfony/CS/Tests/Fixer/Symfony/NewWithBracesFixerTest.php#L81